pkg/route/apis/route/v1/conversion_test.go

```diff
@@ -41,3 +50,108 @@ func TestSupportingCamelConstants(t *testing.T) {
 		}
 	}
 }
+
+func TestRouteSpecConversion(t *testing.T) {
+	scheme := runtime.NewScheme()
+	Install(scheme)
+
+	serviceName := "TestService"
+	serviceWeight := int32(0)
+	var headerNameXFrame string = "X-Frame-Options"
+
+	testcases := map[string]struct {
+		versionedRouteSpec1   *v1.RouteSpec
+		unversionedRouteSpec2 *route.RouteSpec
+	}{
+		"RouteSpec Conversion 1": {
+			versionedRouteSpec1: &v1.RouteSpec{
+				Host: "host",
+				Path: "path",
+				Port: &v1.RoutePort{
+					TargetPort: intstr.FromInt(8080),
+				},
+				To: v1.RouteTargetReference{
+					Name:   serviceName,
+					Weight: &serviceWeight,
+				},
+				TLS: &v1.TLSConfig{
+					Termination:              v1.TLSTerminationEdge,
+					Certificate:              "abc",
+					Key:                      "def",
+					CACertificate:            "ghi",
+					DestinationCACertificate: "jkl",
+				},
+				HTTPHeaders: &v1.RouteHTTPHeaders{
+
+					Actions: v1.RouteHTTPHeaderActions{
```

Comment:
Why do you have a blank line here?
```suggestion
				HTTPHeaders: &v1.RouteHTTPHeaders{
					Actions: v1.RouteHTTPHeaderActions{
```

Reply:
done